ivadomed/uncertainty.py

```diff
@@ -17,41 +17,40 @@ def run_uncertainty(ifolder):
         ifolder (str): Folder containing the Monte Carlo samples.
     """
     # list subj_acq prefixes
-    subj_acq_lst = [f.split('_pred')[0] for f in os.listdir(ifolder)
-                    if f.endswith('.nii.gz') and '_pred' in f]
+    subj_acq_lst = [f.name.split('_pred')[0] for f in Path(ifolder).iterdir()
+                    if f.name.endswith('.nii.gz') and '_pred' in f.name]
     # remove duplicates
     subj_acq_lst = list(set(subj_acq_lst))
     # keep only the images where unc has not been computed yet
-    subj_acq_lst = [f for f in subj_acq_lst if not os.path.isfile(
-        os.path.join(ifolder, f + '_unc-cv.nii.gz'))]
+    subj_acq_lst = [f for f in subj_acq_lst if not Path(ifolder, f + '_unc-cv.nii.gz').is_file()]
```

Comment:
`f` same as above and also the `ifolder` is probably not descriptive enough? not sure if `input_folder` or `image_folder` is more descriptive?